src/scrape_up/dictionary/dictionary.py

```diff
@@ -0,0 +1,50 @@
+from bs4 import BeautifulSoup
+from urllib.request import urlopen
+
+class Dictionary:
+    """
+    Create an instance of the `Dictionary` class.
+    ```python
+    scraper = IMDB()
+    ```
+    | Methods        | Details                                      |
+    | -------------- | -------------------------------------------- |
+    | `.get_word_of_the_day()` | Returns word of the day from Dictionary.com. |
+    | `.word_of_the_day_definition()` | Returns the definition of the word of the day. |
+    """
+    def get_word_of_the_day_url(self):
+        with urlopen("https://www.dictionary.com/") as response:
```

Comment:
Each method also do have their docstring, add them as well with syntax and sample response.